examples/mplot3d/spherical_polygon.py

```diff
@@ -0,0 +1,28 @@
+from mpl_toolkits.mplot3d import proj3d
+from scipy.spatial import ConvexHull
```

Comment:
Looks like our doc-build server does not have scipy installed. Could you remake this without scipy? It doesn't need to look pretty, just needs to demonstrate the feature.